cdp-agentkit-core/cdp_agentkit_core/actions/uniswap_v3/create_pool.py

```diff
@@ -0,0 +1,55 @@
+from cdp import Wallet
```

Comment:
The `cdp` import should be reordered with `make format` its fine for now, but lets add a pre-commit hook + GHA CI check for ruff formatting.

Reply:
hmm it's not getting formatted for some reason with `make format`

```
cdp-langchain-py3.13 ❯ make format
ruff format .
31 files left unchanged
```

Reply:
Added that task to spreadsheet